addons/stock/tests/__init__.py

```diff
@@ -4,3 +4,4 @@
 import test_product
 import test_warehouse
 import test_shipment
+from . import test_stock_location_search
```

Comment:
@Yenthe666 
My example module is `account` module [`account/tests/__init__.py`](https://github.com/odoo/odoo/blob/d77f3a73daa6261adb26ec86d3cffff9141f9bc8/addons/account/tests/__init__.py#L4)

You can find more examples of `relative import` runnin
`rgrep "from . import" ./addons/*/tests --include=__init__.py`

<img width="888" alt="screen shot 2016-05-15 at 11 27 59 am" src="https://cloud.githubusercontent.com/assets/6644187/15275395/24f04270-1a90-11e6-8ca6-6d26c17e61e3.png">